game_data_scraper/game_data_scraper.py

```diff
@@ -31,9 +37,20 @@ def __init__(self, name: str, scraper_urls_raw: list[str], filename: str):
         self.name = name
         super().__init__()
         self.scraper_urls_raw = scraper_urls_raw
-        self.filename = f'file_{filename.split("_")[-1]}'
+        self.filename = filename
+        self.local_save_path = local_save_path
+        self.scraper_type = scraper_type
         print("Completed init")
 
+    def set_filename(self):
+        filename = (
+            f'gameset{self.filename.split("_")[-1].split(".")[0]}'
+            if self.scraper_type == "game"
+            else self.filename.split("_")[0].split(".")[0]
+        )
```

Comment:
Don't love this.  Can we send the filename as it should be to begin with?  Technically the scraper shouldn't even be in charge of saving data (only scraping it), but if we're going to allow it to save data, it shouldn't care what the filename is.